src/invidious/helpers/errors.cr

```diff
@@ -74,6 +74,8 @@ def error_template_helper(env : HTTP::Server::Context, status_code : Int32, exce
   # Don't show the usual "next steps" widget. The same options are
   # proposed above the error message, just worded differently.
   next_steps = ""
+  unfound_tv_box_dark_theme = File.read("assets/404_tv_box_dark_theme.svg")
+  unfound_tv_box_light_theme = File.read("assets/404_tv_box_light_theme.svg")
```

Comment:
Please, don't do that! Instead, put the file URL in the `src` attribute in the HTML, and let the browser and web server properly handle caching.

If you load the image from disk each time you have to show this error, you'll waste all file descriptors the server can have.